MH4U Dex/QuestListTableViewController.m

```diff
@@ -0,0 +1,89 @@
+//
+//  QuestListTableViewController.m
+//  MH4U Dex
+//
+//  Created by Joseph Goldberg on 3/16/15.
+//  Copyright (c) 2015 Joseph Goldberg. All rights reserved.
+//
+
+#import "QuestListTableViewController.h"
+
+#import <CoreData/CoreData.h>
+
+#import "CoreDataController.h"
+
+#import "Quest.h"
+
+#import "QuestListTableViewCell.h"
+
+typedef NS_ENUM(NSInteger, StarSections) {
+    OneStarSection = 0,
+    TwoStarSection = 1,
+    ThreeStarSection = 2,
+    FourStarSection = 3,
+    FiveStarSection = 4,
+    SixStarSection = 5,
+    SevenStarSection = 6,
+    EightStarSection = 7,
+    NineStarSection = 8,
+    TenStarSection = 9,
+    
+    StarSectionCount
+};
+
+@interface QuestListTableViewController ()
+
+@property (nonatomic, strong) NSMutableArray *questMasterArray;
+
+@end
+
+@implementation QuestListTableViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    
+    
+    CoreDataController *coreDataController = [CoreDataController sharedCDController];
```

Comment:
It looks like you're only using this for the `.managedObjectContext` property—if so, it might make sense to have an `NSManagedObjectContext` variable and store the value of that property in it straight away, rather than storing the `CoreDataController`.